README.md

```diff
@@ -25,11 +25,12 @@ Go to the [documentation directory](docs/).
 
 ### Setup
 
-Make sure you have your [_$GOPATH_ environment variable set](https://golang.org/doc/code.html#GOPATH).
+Make sure you have your [_$GOPATH_ environment variable set](https://golang.org/doc/code.html#GOPATH). $GOPATH should be
+set to the terraform-provider-softlayer directory.
 
 _$GOPATH/bin_ should also be in your _$PATH_ (e.g. `export PATH=$GOPATH/bin:PATH`).
 
-Now you need to get the main dependencies.
+To retrieve dependencies run these commands from the root directory of terraform-provider-softlayer.
```

Comment:
OK, so that's uncovered some confusion which is good! Err, let's see so when I clone this project down ...

1. I need a GOPATH, that's setup as explained in the linked documentation.
2. I need $GOPATH/bin on my path.

`go get` should install the dependencies to my GOPATH.

I'll use the make scripts to compile binaries, and then point terraform to those binaries in order to work with it?